angular/projects/spark-angular/src/lib/components/sprk-masthead/components/sprk-masthead-selector/sprk-masthead-selector.component.ts

```diff
@@ -57,15 +55,15 @@ import { ISprkMastheadSelectorChoice } from '../sprk-masthead-selector/sprk-mast
             href="#"
           >
             <span
-              class="sprk-c-Masthead__selector-dropdown-title sprk-b-TypeBodyTwo sprk-o-Stack__item sprk-o-Stack__item--flex@xxs"
+              sprkStackItem
+              sprkText
+              variant="bodyTwo"
+              class="sprk-c-Masthead__selector-dropdown-title sprk-o-Stack__item--flex@xxs"
               >{{ heading }}</span
             >
             <sprk-icon
               [iconName]="triggerIconName"
-              additionalClasses="
-                sprk-c-Icon--toggle
-                sprk-Stack__item
-              "
+              additionalClasses="sprk-Stack__item sprk-c-Icon--toggle"
```

Comment:
This can't use our stack item directive because we don't want the stack item class on the `<sprk-icon>` custom element. We want to pass it in through additional classes so it ends up on the svg element.